packages/grafana-data/src/types/templateVars.ts

```diff
@@ -54,7 +54,7 @@ export interface AdHocVariableFilter {
   operator: string;
   value: string;
   values?: string[];
-  origin?: 'dashboard';
+  origin?: 'dashboard' | string;
```

Comment:
still need string here since AdHocFiltersWithLabels extends this in scenes and there would be a type mismatch